charts/prometheus-stackdriver-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Stackdriver exporter for Prometheus
 name: prometheus-stackdriver-exporter
-version: 1.8.3
+version: 1.8.4
```

Comment:
Thanks for your PR. As it is a new feature can you pls. increase the minor version.

Reply:
@rpahli done https://github.com/prometheus-community/helm-charts/pull/1051/commits/bbd4a023df4c9d2e9399f61ea5908205683f9b61